wordpress-mobile/WordPress-iOS
# Migrate People Remote to NSURLSession api.

Refs #5245 

To test:
 - Log to WP.com account
 - Open Blog->Team
- Check if the list of users of the blog is being displayed ok.



Needs review: @jleandroperez
